data/campaigns/Eastern_Invasion/ai/ca_ogres_flee.lua

```diff
@@ -42,24 +42,25 @@ function ca_ogres_flee:execution()
                 local rating = - dist
 
                 -- If we can reach the edge, we do so
-                if (dist == 0) then
-                    rating = rating + 1000
-                end
+                if (dist == 0) then rating = rating + 1000 end
+
+                local enemy_weight = 0.5
+                local enemy_rating = - (enemy_attack_map.units:get(r[1], r[2]) or 0) * enemy_weight
 
                 local enemy_rating = 0
```

Comment:
CI check flags a warning against `enemy_rating` being defined and then redefined to 0.